src/components/03-templates/Jarjestaja/LupapaatoksetTable/index.js

```diff
@@ -192,6 +192,7 @@ export default function LupapaatoksetTable({
           : common.paatoskirje
       ),
       hidden:
+        koulutusmuoto.koulutustyyppi === koulutustyypitMap.LUKIO ||
```

Comment:
Tässä piilotetaan väärä kolumni. Pitäisi olla samalla tavalla kuin perusopetuksellakin. Näillä on periaatteessa päätöskirje se lupa pdf ja näin ollen niille pitää olla uudelleen nimetty kolumni joka viittaa tuohon päätöskirjeeseen. Katso paikat jossa on koulutusmuoto.koulutustyyppi === koulutustyypitMap.ESI_JA_PERUSOPETUS ja lisäät näihin lukion myös ehdoksi. Sitten pitäisi mennä oikein.